### ISSUE TITLE
My approach to decision making - radio buttons do not read well

### ISSUE THREAD

**Issue by gobyrne** [GCTC-NTGC/TalentCloud#50]
The "My approach to decision making display on the manager profile needs a redesign. It currently reads very poorly on the screen reader but is also confusion in general. The radio button input table should be replaced by simple sentences that describe the approach.
